building-a-preview/index.md

```diff
@@ -0,0 +1,777 @@
+- [Add additional/multiple Base Previews](#add-additionalmultiple-base-previews)
+- [Stop using the current Base Preview and set a new Base Preview](#stop-using-the-current-base-preview-and-set-a-new-base-preview)
+
+#### Set a Base Preview for the first time
+
+Ready to get started with your first Base Preview? Check out:
+[How to set a Base Preview](#how-to-set-a-base-preview).
+
+#### Add additional/multiple Base Previews
+
+Want to use more than one Base Preview? Take a look at:
+[Add multiple Base Previews](#add-multiple-base-previews).
+
+#### Stop using the current Base Preview and set a new Base Preview
+
+Want to change the Base Preview you're using? First, you'll need to have a
+[Preview build](#build-a-preview) ready to set as your new Base Preview. Then:
+
+1. Go to the **Manage Base Previews** link on the Repository Dashboard;
+2. Click the checkbox next to the Base Preview you want to stop using to
+   deselect it;
+3. Click the checkbox next to the Preview you want to set as your new Base
+   Preview;
+4. Press the **OK** button.
+
+The Preview you've deselected will move out of the Base Previews section of the
+dashboard, and the new Base Preview will appear here, instead.
+
+![Deselect and select new Base Preview](_images/deselect_and_select_new_base_preview.png)
+
+### Add multiple Base Previews
+
+Want to use more than one Base Preview? No problem! Simply follow the
+instructions in [How to set a Base Preview](#how-to-set-a-base-preview), and
+check the checkboxes next to all of the Previews that you want to use as Base
+Previews.
+
+![Set multiple Base Previews](_images/set_multiple_base_previews.png)
+
+Keep in mind that when you've selected multiple Base Previews, every new Preview
+build (including automated builds from pull requests) will create a build from
+_each_ Base Preview. In my sample project, I've set two base Previews, and
+building a Preview from a new PR automatically created two Previews.
+
+![Multiple Base Previews generating multiple Preview builds](_images/multiple_base_preview_preview_builds.png)
+
+### Stop using a Base Preview
+
+If you want to stop using a Base Preview, simply:
+
+1. Go to the **Manage Base Previews** link on the Repository Dashboard;
+2. Click the checkbox next to the Base Preview you want to stop using;
+3. Press the **OK** button.
+
+The deselected Preview will disappear from the Base Preview section of the
+dashboard, and subsequent Preview builds - including automated builds from git
+provider integrations - will no longer start from that Base Preview.
+
+### Keeping Base Previews up-to-date
+
+If you want to keep your Base Previews up-to-date, there are two ways you can do
+it:
+
+- [Automatically update Base Previews](#automatically-update-base-previews)
+- [Manually update Base Previews](#manually-update-base-previews)
+
+#### Automatically update Base Previews
+
+You'll generally want to keep your Base Preview up to date with your latest
```

Comment:
> You'll generally want to keep your Base Preview up to date with your latest
 codebase, and a fresh copy of your database, image files, and other assets

Move this to the parent section